docs/source/project-structure.rst

```diff
@@ -1,6 +1,9 @@
 Project structure
 =================
-TBD
+
+The **Leapp Project** provides the framework for automating in-place upgrades from one major version of Red Hat Enterprise Linux to another. It is organized into a set of modular components, each responsible for specific tasks during the upgrade process. These components are managed across multiple **repos**, logically grouped to facilitate upgrades between different RHEL paths.
+
+This documentation explains the structure of the **Leapp repository** and provides guidance on how to determine where to place new entities, such as actors, models, and libraries, within the project.
```

Comment:
```suggestion
This documentation explains the structure of this git repository and provides guidance on how to determine where to place new entities, such as actors, models, and libraries, within the project.
```

Well, the name is not the best one, but we have to live with it. Phrase `the leapp repository` would be speaking in current terminology about one specific leapp repository (e.g. system_upgrade_el7toel8) or the actual leapp git repositoriy. So let's use either
* of this project
* of this git repository
* of the leapp-repository git repository